src/style.css

```diff
@@ -1 +1,53 @@
 /* Styles go here */
+
+.header {
+  margin: 0;
+  padding: 0 50px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.logo {
+  padding: 10px;
+}
+
+.nav_list {
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+}
+
+.nav_item {
+  margin: 0 10px;
```

Comment:
your last `.nav_item` doesn't need the margin from the right 😉

Reply:
Question : I added the class nav_item_last , so why margin-left: 10px doesnt work but margin-right: 0 worked out ?
Thanks !

Reply:
didn't get the question 🤔

Reply:
i can advise you using pseudoclass `:not(:last-child)`